### DISCUSSION TITLE
Compile of tasmota32s3-qio_opi-all environment fails (no code modification)

### DISCUSSION THREAD

**Discussion by msjfb** [arendst/Tasmota#22686] [Support request]
Hi, Can anyone succeed in compiling this ? Other users in Discord have reproduced and confirmed the problem, but I have yet to find a solution...
![Tasmota Compile fail](https://github.com/user-attachments/assets/ea7bd0ff-2bc6-4ec3-9459-f7a71fe7bcc3)
 Any help in trying to solve this will be greatly appreciated.

**Comment by arendst** [comment]
I see where it goes wrong.

You probably enabled `define USE_WEBCAM`. That should be fine if handling `define ENABLE_RTSPSERVER` would have been configured correctly (by the programmer). The latter is not the case.

As a quick fix enable `define ENABLE_RTSPSERVER` too and your compile will finish correctly.

I will make a change to the webcam code to fix compiling with only `define USE_WEBCAM` now.

**Comment by arendst** [comment]
Fixed in latest dev.

**Reply by msjfb** [reply]
Super, Thank you very much!  'tasmota32s3-qio_opi-all' now compiles without error.
Now on to try installing Tasmota on XIAO ESP32-S3 SENSE : many errors during compile when using the information found here:
https://templates.blakadder.com/seeedstudio_XIAO_ESP32S3_SENSE.html 
I will keep digging. Thanks again.
